documents/charter.md

```diff
@@ -1,23 +1,24 @@
-Video traffic is already 70% of the overall traffic volume on the Internet and is expected to grow to 80% by 2028.
+Video traffic is 70% of the overall traffic volume on the Internet and is expected to grow to 80% by 2028.
 Across developed and emerging markets video traffic forms 50-80% of traffic volume on mobile networks.
 New formats like short form videos have seen tremendous growth in recent years.
 These growth trends are likely to increase with new populations coming online on mobile-first markets.
 
-Mobile network operators continuously invest in network resources, including deployment of new generations or new bands of spectrum.
-Since spectrum is a limited and expensive resource, operators often make use of flow-based traffic handling such as shaping of video traffic, especially when the network is highly loaded.
-Operators cannot explicitly measure the degradation that shaping causes to end user quality of experience (QoE), making this approach open loop. 
+While mobile network operators continuously invest in new network capacity, including deployment of new generations or new bands of spectrum, these efforts are not keeping pace with growth in demand. These network operators have found it faster and less expensive to invest in shaping of video traffic on a per-flow basis, which negatively affects video stream quality. Because that traffic is encrypted, network operators cannot explicitly measure the degradation that shaping causes to end user quality of experience (QoE), making this approach open loop. 
```

Comment:
IME (multinational cellular operator) the network traffic shapers  are typically TCP only. If that's the case for other operators then that becomes another motive for a SCONE approach, as most of the major video providers are QUIC and hence unaffected by the traffic shapers.

Reply:
I think perhaps this may be the crux of the issue, which may merit calling out in the text. Such as: 

> MNOs have historically utilized traffic shaping of video streaming, which was TCP-based. Now that QUIC is seeing widespread adoption, that traffic shaping is no longer possible due to the way that QUIC has been designed (in part to reduce or eliminate the effects of middlebox functions).

Reply:
> IME (multinational cellular operator) the network traffic shapers are typically TCP only. If that's the case for other operators then that becomes another motive for a SCONE approach, as most of the major video providers are QUIC and hence unaffected by the traffic shapers.

@Kevsy this is not true and has not been true for years. Let me assure you that shapers are perfectly capable of identifying video traffic running over QUIC and shaping it. We see this in numerous markets. A reminder that our mobile video traffic to apps is 99% QUIC.

Reply:
> @Kevsy this is not true and has not been true for years. Let me assure you that shapers are perfectly capable of identifying video traffic running over QUIC and shaping it. We see this in numerous markets. A reminder that our mobile video traffic to apps is 99% QUIC.

@mjoras as I say 'IME', so thanks for the clarification - possibly it's more widespread in certain markets. In any case there (again, IME) is a widespread deployment of TCP optimizers which become increasingly irrelevant as QUIC dominates video, so still worth mentioning that as a motive as Jason suggests above.

> The reason we are focusing on QUIC is rather that this problem is easier to solve for QUIC, and it is the most important protocol to solve it for given its dominance for this kind of video traffic.

Yep, agreed